routing/routing.xml

```diff
@@ -549,7 +561,15 @@
 			<select value="-1" t="mtb:scale" v="5"/>
 			<select value="-1" t="mtb:scale" v="6"/>
 
-			<select value="-1" t="vehicle" v="no"/>
+			<select value="1" t="vehicle" v="no"/>
+			<!-- The tag vehicle=no is normally used when mappers
+			want to exclude motor vehicles. I have never seen a case
+			when such a road was actually forbidden for bicycles as
+			well. (Mappers often do not bother to add the bicycle=yes).
+			Example:www.openstreetmap.org/way/159679495  This is
```

Comment:
Yes, you are right concerning the definition. The problem is, that wrong tagging is the norm with vehicle=no. Maybe this is because in everyday language, the term "vehicle" is not used for bicycles at all. If you do a picture search for "vehicle" it only produces cars, lorries, tanks etc

When you go with a bike, you can always dismount. So maybe we can allow vehicle=no here and give it a factor of 0.4 later?